packages/veto-app/backend/src/utils/config.ts

```diff
@@ -44,6 +44,9 @@ export type Config = BaseConfig & {
       keyId: string
     }
   }
+  db: {
+    file: string
```

Comment:
is it the file itself or the file location?

Reply:
Good point, looks like it's a fileurl

Reply:
@JoTrdl would be great to have this information in the name. `filePath` or `pathToDatabase` should do the trick!

It will help when we encounter the variable without context.

Reply:
Updated to `filePath`